### DISCUSSION TITLE
MacOS very low contrast for pan+reverb slider

### DISCUSSION THREAD

**Discussion by pieterbos** [jamulussoftware/jamulus#1273] [UI Related]
I downloaded Jamulus 3.7.0-RC1, and noticed the new fancy UI. Looks a lot better!

However, I see some usability/accessibility problems with the new pan and reverb sliders. They used to be high contrast and very clear. They look really clean now, but are very low contrast and hard to see.
This applies to the sliders themselves, which are almost the same color as the background now, and in particular to the tick marks. Those are very useful to quickly set these to the same level as before in the same situation, but are now hardly visible anymore. I have normal vision and am on my high contrast IPS macbook screen and I can hardly see them - with slightly worse vision or less good displays those things will just be invisible.

I did only try this on Mac OS, I'm not sure if the issue is present on all supported systems.

I can understand why these two slides are made to look less important than the volume sliders. But I think there could be something in between, perhaps it could be good to make at least the tick marks a little bit higher contrast?

**Comment by dcorson-ticino-com** [comment]
Hi Pieter,
On Win10 the pan and reverb sliders are blue just as before and the rest of the indications are all high contrast as they always were.
The only thing that has changed that I can see is that the background is "smoother".
Do you see less contrast with the other skins too?

**Comment by pieterbos** [comment]
This is how the three skins look on my MacOS Big Sur macbook pro:

<img width="929" alt="Screenshot 2021-03-15 at 14 20 20" src="https://user-images.githubusercontent.com/21126/111159760-bd753e00-8599-11eb-8fc0-2a8e02385b07.png">
<img width="924" alt="Screenshot 2021-03-15 at 14 20 35" src="https://user-images.githubusercontent.com/21126/111159754-bc441100-8599-11eb-9933-f9ff8fba6a93.png">

<img width="923" alt="Screenshot 2021-03-15 at 14 20 46" src="https://user-images.githubusercontent.com/21126/111159746-b9e1b700-8599-11eb-8f9f-c56483df589d.png">

**Reply by pieterbos** [reply]
Oh I just checked by reverting to 3.6.2, and indeed the sliders are the same in that version - while they are blue and high contrast on Linux. That makes this an entirely different issue - pan/reverb sliders are low contrast on OS X :)

**Comment by dcorson-ticino-com** [comment]
Here on Win10
![V370](https://user-images.githubusercontent.com/64357360/111161224-31641600-859b-11eb-96fb-0ca91821c3ff.JPG)
